src/main/java/core/basesyntax/MainApp.java

```diff
@@ -1,5 +1,17 @@
 package core.basesyntax;
 
 public class MainApp {
+    public static void main(String[] args) {
 
+        Machine truck = new Truck();
+        Machine bulldozer = new Bulldozer();
+        Machine excavator = new Excavator();
+        Machine[] cars = {truck, bulldozer, excavator};
```

Comment:
Variables should be introduced when they are reused in different places 
or when we give understandable name for complicated code part and thus increasing readability.
This is not the case, used only once, not more clear than command under variable
```suggestion
        Machine[] cars = {new Truck(), new Bulldozer(), new Excavator()};
```